### DISCUSSION TITLE
Where to find the admin token?

### DISCUSSION THREAD

**Discussion by jaymc-arg** [zitadel/zitadel#6190] [Q&A]
Hi!
I have a newbie question. I can't find anywhere the Bearer Token to make [this](https://zitadel.com/docs/apis/resources/mgmt/management-service-import-human-user) request. 
I'm a little bit lost with the steps, user types and tokens provided by zitadel.
Does that operation can be done by a human user or must be done by a machine user? Do human users have tokens?
Thank you all.

**Comment by hifabienne** [comment]
Hei @jaymc-arg 
The request can be done by a human or a service user.
To access the ZITADEL APIs a user needs some role to do that.
The ZITADEL internal roles are called managers. And you can give your users the roles on different levels (Instance, Organization, Project)
https://zitadel.com/docs/guides/manage/console/managers
For example you can give your user on the organization where he should be able to create new users the role ORG_MANAGER. Then the user is able to do all the requests tor that organization.

**Reply by jaymc-arg** [reply]
I've already tried assigning every available role (even ROLE_MANAGER), but it keeps giving back this error:
```
[
  {
    '@type': '[type.googleapis.com/zitadel.v1.ErrorDetail](http://type.googleapis.com/zitadel.v1.ErrorDetail)',
    id: 'AUTH-5mWD2',
    message: 'No matching permissions found'
  }
]
```
This is the data i'm sending in the request:
```
async function registerZitadel(req,res){
    let data = { 
    "userName": "minnie-mouse",
    "profile": {
      "firstName": "Minnie",
      "lastName": "Mouse",
      "nickName": "Mini",
      "displayName": "Minnie Mouse",
      "preferredLanguage": "en",
      "gender": "GENDER_FEMALE"
    },
    "email": {
      "email": "[minnie@mouse.com](mailto:minnie@mouse.com)",
      "isEmailVerified": true
    }
    ]
  }

```
For the record, i'm logged in as the ZITADEL Admin user thats starts with the app, and gave him every role "Self Management Global".
Something i'm not seeing?

**Reply by hifabienne** [reply]
Hei @jaymc-arg 
I would try it with the ORG_OWNER role, as this is for sure allowed to do the call.
Next to check is the organization on which you perform the request. So the ORG_MANER only has permissions in the Organization, where you give the role. If you want to enable for all organization you should go to the instance level and give him the IAM_OWNER role.
So please ensure that the user has the role for the Organization you like. Per default the requests are performed on the organization where the user belongs to, if you need another organization please send along the Organization Header. x-zitadel-orgid with the id of the organization you like

**Reply by codekoala** [reply]
@jaymc-arg did you ever figure out how to get past the `No matching permissions found` error?
